cylc/flow/network/resolvers.py

```diff
@@ -629,6 +629,8 @@ async def subscribe_delta(
                     del delta_queues[w_id][sub_id]
             if sub_id in self.delta_store:
                 del self.delta_store[sub_id]
+            if sub_id in self.delta_processing_flows:
+                del self.delta_processing_flows[sub_id]
```

Comment:
@dwsutherland, do you know if it makes sense to delete the entry here or should be cleared elsewhere in the code?